Android/app/src/main/java/com/example/yello/common/Adapter.java

```diff
@@ -0,0 +1,63 @@
+package com.example.yello.common;
+
+import android.content.Context;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class Adapter extends PagerAdapter {
+
+    private Context context;
+    private LayoutInflater inflater;
+    private Integer [] images = {R.drawable.coffee, R.drawable.cooking, R.drawable.wine}; // image of classes
+    private String [] classes = {"Coffee", "Cooking", "Wine tasting"};
```

Comment:
These should be supplied from the server, correct? If this is a mock data, please leave a comment & add a TODO to remove it later when the backend responses are available.

Reply:
Yeah, Those should be changed later. I will add comment.